src/components/SignupForm/index.js

```diff
@@ -37,44 +37,43 @@ const Signup = props => {
 
   return (
     <div>
-      <div className="signStr spacebottom">Create an account</div>
+      <div className='signStr spacebottom'>Create an account</div>
       <form onSubmit={handleSubmit}>
         <InputBox
-          type="text"
+          type='text'
           value={signupData.username}
-          name="username"
-          placeholder="USERNAME"
+          name='username'
+          placeholder='USERNAME'
           onChangeInput={e => onChangeUsername(e)}
         />
         <InputBox
-          type="email"
+          type='email'
           value={signupData.email}
-          name="email"
-          placeholder="EMAIL"
+          name='email'
+          placeholder='EMAIL'
           onChangeInput={e => onChangeEmail(e)}
         />
         <InputBox
-          type="password"
+          type='password'
           value={signupData.password}
-          name="password"
-          placeholder="PASSWORD"
+          name='password'
+          placeholder='PASSWORD'
           onChangeInput={e => onChangePassword(e)}
         />
         <InputBox
-          type="password"
+          type='password'
           value={signupData.confirm_password}
-          name="confirm-password"
-          placeholder="CONFIRM PASSWORD"
+          name='confirm-password'
+          placeholder='CONFIRM PASSWORD'
           onChangeInput={e => onChangeConfPassword(e)}
         />
         {signupData.error !== '' ? (
```

Comment:
`signupData.error` can also be any falsy value aside from an empty string. Your check should take care of that also